servers/parser-graphql-wrapper/src/datasources/ParserAPI.ts

```diff
@@ -146,7 +146,7 @@ export class ParserAPI extends RESTDataSource {
   private parserResponseToItem(parserResponse: ParserResponse): Item {
     return {
       itemId: parserResponse.item_id,
-      id: IntMask.encode(parserResponse.item_id),
+      id: createItemId(parserResponse.item_id),
```

Comment:
@collectedmind I know we have a million ids... should we add `readerViewId` to make it more obvious what this is...